bower.json

```diff
@@ -19,22 +19,45 @@
   "homepage": "https://github.com/PolymerElements/paper-dialog",
   "ignore": [],
   "dependencies": {
-    "neon-animation": "PolymerElements/neon-animation#^1.0.0",
-    "paper-dialog-behavior": "PolymerElements/paper-dialog-behavior#^1.0.0",
-    "iron-overlay-behavior": "PolymerElements/iron-overlay-behavior#^1.7.0",
-    "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "polymer": "Polymer/polymer#^1.1.0"
+    "neon-animation": "PolymerElements/neon-animation#2.0-preview",
+    "paper-dialog-behavior": "PolymerElements/paper-dialog-behavior#2.0-preview",
+    "iron-overlay-behavior": "PolymerElements/iron-overlay-behavior#2.0-preview",
```

Comment:
In the 1.x variant we specify the version to 1.7.0, so I added it here for "parity" (even if not needed), since it caused some confusion in other PRs